### ISSUE TITLE
disable spell module by default

### ISSUE THREAD

**Issue by Konfekt** [vim-pandoc/vim-pandoc#301]
The spell module is better customized by the user through an auto-command such as
```
autocmd Filetype pandoc set spell spelllang=en,it
```
instead of via a module that doubles this functionality and is enabled by default.

**Comment by fmoralesc** [comment]
I disagree on this point. It is a good idea to have convenient defaults and not ask the users to add extra configuration. We can make the module more flexible if needed. People who don't want it can always disable it.

We could try to detect if the user has set up a custom rule beforehand, though.

**Comment by Konfekt** [comment]
Okay, this is a maintainer's preference. This suggestion was inspired by other plug-ins such as `vimtex` for editing TeX files. Many users appreciate a spellcheck when editing TeX files, but the plug-in does not even offer any documentation on how to do so. Probably it is understood that a Vim user ought to know how to so (for example, by a filetype autocmd or adding `set spell` to `/ftplugin/tex.vim`).

**Comment by fmoralesc** [comment]
> Probably it is understood that a Vim user ought to know how to so

I don't like to to make those assumptions. [Eternal September](https://en.wikipedia.org/wiki/Eternal_September) and all.

**Comment by Konfekt** [comment]
Fair enough. The aloof Eternal September attitude may be behind that, but instead of a module `spell`, enabled by default, adding a FAQ entry would double less functionality:

Q: On opening a `pandoc` file the spell check is not enabled, what's wrong?

A: Add to `vimrc` the line
```vim
autocmd FileType pandoc setlocal spell
```

That said, I have no strong opinions on it; this suggestions just seems more efficient and flexible to me.

**Comment by fmoralesc** [comment]
Maybe. I think in this case (unlike with the `chdir` module) having the default module is the best option for the general case, since I assume most people will want to add spelling configuration for working on markdown files.

**Comment by aconitumnapellus** [comment]
Sorry to hijack this issue, but i would like to disable this module so it does not load automatically.
How do i approach this?

**Comment by fmoralesc** [comment]
Should add this to your init file: 

```
let g:pandoc#modules#disabled = ["spell"]
```

See `:help vim-pandoc-modules`

**Comment by fuadnafiz98** [comment]
👀 **This worked for me**

``` 
let g:pandoc#spell#enabled=0
```
